Trust and Compliance: Fiscal provisions (#105)
pretix/pretix-docs: docs/trust/fiscal/hungary.md

# Hungary

{% include "warning-tax-law.md" %}

## Value-added tax (VAT)

> When selling goods and services in Estonia, VAT ("Általános forgalmi adó", "ÁFA") may apply.

Reviewer: ```suggestion
When selling goods and services in Hungary, VAT ("Általános forgalmi adó", "ÁFA") may apply.
```

You can configure all Estonian VAT rates in pretix using [tax rules](../../guides/taxes.md), including more complex tax cases (e.g. intra-EU reverse charge transactions).

## Invoicing

### E-invoicing

Hungary has complex legal requirements around sending invoices, including an obligation for real-time invoice reporting.
pretix does **not** support this, but it is possible to sell tickets through pretix and send and report invoices outside of pretix.

More information is available on the website of the European Commission: [eInvoicing in Hungary](https://ec.europa.eu/digital-building-blocks/sites/display/DIGITAL/eInvoicing+in+Hungary).

## pretixPOS

> When operating a cash register in Hungary, the cash register must integrate with a fiscal device and certified conforming to [local regulations](https://njt.hu/jogszabaly/2025-8-20-2X).

Reviewer: ```suggestion
Cash registers operated in Hungary must be certified to comply with [local regulations](https://njt.hu/jogszabaly/2025-8-20-2X).
Cash registers also have to interface with a fiscal device. 
```
